Core/src/Material/SurfaceMaterialMapper.cpp

```diff
@@ -247,16 +246,20 @@ void Acts::SurfaceMaterialMapper::mapMaterialTrack(
 
   // Assign the recorded ones, break if you hit an end
   while (rmIter != rMaterial.end() && sfIter != mappingSurfaces.end()) {
-    if (volIter != mappingVolumes.end() && encounterVolume == true &&
+    // Material not inside current volume
+    if (volIter != mappingVolumes.end() &&
         !volIter->volume->inside(rmIter->position)) {
-      encounterVolume = false;
-      // Switch to next material volume
-      ++volIter;
+      double dist_vol = (volIter->position - mTrack.first.first).norm();
```

Comment:
Can you use CamelCase, i.e. `distVol`, etc. ?

Reply:
Yes you are right. Done.